### DISCUSSION TITLE
Priors for gene kinetics fit

### DISCUSSION THREAD

**Discussion by mxposed** [theislab/scvelo#664] [Q&A] [answered]
In the paper in the methods you have a section “Accounting for insufficiently observed kinetics with prior information”.

I wonder if the current API of scVelo allows to specify this prior information about the root cells and where would that be?
Is is the `steady_state_prior` parameter to `recover_dynamics`?

**Comment by WeilerP** [comment, marked as answer]
@mxposed, sorry, this feature is currently not yet available in scVelo.

**Reply by mxposed** [reply]
Thank you! Are there any plans on adding this? Is there a manual workaround to implement this, maybe?

**Reply by WeilerP** [reply]
There is #144 that attempted to implement this. The code base has evolved significant, however, and will continue to do so since then. I'm planning to first unit test scvelo before adding any major features (hope to be done with it rather soon). Thinking about adding this feature in a couple of weeks. I am just unsure if it will not lead to a false sense of security TBH, since you can steer the system towards your expected output even though the phase portraits alone do not offer enough info. This is e.g. the case for linear or very noisy (blobs) phase portraits.

**Reply by mxposed** [reply]
Oh, I think you're totally right, I understand your concern. Looking at the paper, however, I think that it will be useful for some datasets,  to fine-tune the already good fit of velocity. It seems like an “expert-level” API

**Reply by WeilerP** [reply]
Yes, agreed that it can be useful for some datasets. Currently working on an approach to ideally automatically resolve ambiguity in the phase portraits.
